stl/inc/xmemory

```diff
@@ -776,6 +776,126 @@ _NODISCARD constexpr allocation_result<typename allocator_traits<_Alloc>::pointe
 // The number of user bytes a single byte of ASAN shadow memory can track.
 _INLINE_VAR constexpr size_t _Asan_granularity = 8;

```

Comment:
Since `_Asan_granularity - 1` is used more than a couple of times, I'm wondering if it makes sense to add a named variable for readability?